### PR TITLE
Miscellaneous

### PR DESCRIPTION
**Pull request by GabrielaReyna**

### CODE REVIEW

File: src/components/ProductNav.astro [hunk 1]
```diff
@@ -39,7 +42,15 @@ const getStartedHref = nav.entries.filter(
 		<div class="flex items-center gap-4 ml-auto flex-none text-stone-200">
 			<a href={DISCORD_URL} class="flex items-center gap-1.5 hover:text-white">
 				<Icon name="fa-brands:discord" class="w-5 h-5" />
-				<span class="sr-only md:not-sr-only">Community</span>
+				<span class="sr-only md:not-sr-only"></span>
+			</a>
+			<a
+				href={product.github}
+				class="rounded-md flex items-center bg-stone-700 p-1 gap-1.5 hover:text-white"
```
Comment (bates64) [inline src/components/ProductNav.astro:49]: I'm not fully sold on the background. Perhaps a translucent border would be better?

image

**Review by bates64 — CHANGES_REQUESTED** [review]

File: src/content/docs/cheerpj2/index.mdx [hunk 2]
```diff
@@ -13,10 +13,18 @@ CheerpJ is a **drop-in replacement** for the JVM. It has extensive compatibility
 
 With CheerpJ, you can:
 
-- Run existing **Java applications** in the browser with no changes
-- Include **Java applets** in webpages without legacy plugins
-- Migrate **Java Web Start / JNLP** applications to work on modern systems
-- Use Java libraries in JavaScript/TypeScript seamlessly
+- Run existing [Java applications] in the browser with no changes
+- Include [Java applets] in webpages without legacy plugins
```
[the remark this answers is not in the document]
Comment (GabrielaReyna) [reply src/content/docs/cheerpj2/index.mdx:17]: agree

File: src/content/docs/cheerpj3/12-reference/00-cheerpjInit.md [hunk 3]
```diff
@@ -358,6 +359,20 @@ cheerpjInit({
 });
 ```
 
+### `licenseKey`
+
+```ts
+licenseKey?: string;
+```
+
+This option expects a license key string. The non-commercial license message will be removed from the CheerpJ display if a valid license key is used.
```
[the remark this answers is not in the document]
Comment (GabrielaReyna) [reply src/content/docs/cheerpj3/12-reference/00-cheerpjInit.md:368]: Good idea

File: src/components/ProductNav.astro [hunk 4]
```diff
@@ -16,6 +16,9 @@ const nav = await getProductNav(product.id);
 const getStartedHref = nav.entries.filter(
 	(entry) => entry.type === "directory",
 )[0]?.href;
+const githubData = await fetch(
+	"https://api.github.com/repos/leaningtech/" + product.repositoryName,
+).then((response) => response.json());
```
[the remark this answers is not in the document]
Comment (GabrielaReyna) [reply src/components/ProductNav.astro]: shall we not show the GitHub CTA at all if the response is not ok? For now, I just removed the star icon if unsuccessful.

File: src/components/ProductNav.astro (hunk 1 above)
Comment (GabrielaReyna) [reply src/components/ProductNav.astro]: I have pushed a different style

File: src/components/ProductNav.astro (hunk 4 above)
[the remark this answers is not in the document]
Comment (bates64) [reply src/components/ProductNav.astro]: I like how you've done it now :+1:

**Review by bates64 — CHANGES_REQUESTED** [review]
Almost LGTM; it's a shame about the star count not being accessible for whatever reason, but no huge deal. The Discord and GitHub buttons have different hover states - can you remove the background from the GitHub button entirely?

**Comment by GabrielaReyna** [comment]
stars only load for Cheerp in the preview build, I suspect about the GH rate limit.

**Pull request merged**